reference-tests/src/test/java/net/consensys/linea/ReferenceTestWatcher.java

```diff
@@ -45,4 +52,22 @@ public void testFailed(ExtensionContext context, Throwable cause) {
     MapFailedReferenceTestsTool.mapAndStoreFailedReferenceTest(
         testName, logEventMessages, JSON_OUTPUT_FILENAME);
   }
+
+  @Synchronized
+  private static ch.qos.logback.classic.Logger getLogbackLogger() {
+    try {
+      org.slf4j.Logger slf4jLogger = null;
+      for (int i = 0; i < LOGBACK_POLL_ATTEMPTS; i++) {
```

Comment:
Why are these attempts needed? Is there way to alter the instantiating order of this Test watcher and the root logger so that the root logger will be available before the test watcher is instantiated.

Reply:
This was added because there were occasions when `substituteLogger` was being retrieved. Since adding this I have excluded some jars from conflicting dependencies so I'll test whether this is still required.

Reply:
It looks like it is still required.